pkg/proxy/winkernel/proxier.go

```diff
@@ -722,9 +723,28 @@ func NewProxier(
 		if err != nil {
 			return nil, err
 		}
+
+		/* Source VIP is required for overlay network mode. KubeProxy will determine the Source Vip in following order:
+		1. Use the Source Vip in the KubeProxy winkernel configuration options
+		2. If the Source Vip is not provided in the winkernel configuration, check if an HNS endpoint exists with the well-known name
+		   It is the responsibility of the CNI to ensure the Source Vip is created using the well known-documented name
+		3. If the expected Source Vip endpoint name does not exist on the host, KubeProxy will attempt to create the Source Vip
+		*/
```

Comment:
use `//`-style comments please. We pretty much only ever use `/*` style for the top-of-file comments (license etc).